ouroboros-consensus/src/Ouroboros/Consensus/Node/Tracers.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE StandaloneDeriving #-}
+
```

Comment:
Can you align the `#-}`? We use `stylish-haskell` which does this for us. The next we touch this file, we'd automatically make this change.

Can you can remove the new empty line below as well?

Reply:
Ah, you removed. This is no longer relevant.